services/libs/integrations/src/integrations/premium/hubspot/api/utils/getOrganizationDomain.ts

```diff
@@ -0,0 +1,3 @@
+export const getOrganizationDomain = (website: string) => {
+  return website.replace('https://', '').replace('http://', '').replace('www.', '').replace('/', '')
```

Comment:
It's better to use `new URL(website).host` to do this. If the website is spelled like `https://crowd.dev/en`, it breaks.

Reply:
updated :+1: